src/services/RestDataAPI.ts

```diff
@@ -0,0 +1,50 @@
+import { DEFAULT } from '../constants/constants';
+import { LOCAL_STORAGE_KEY } from '../constants/localStorage';
+import { KeyOfCategory, KeyOfSortingKey, Restaurant } from '../types/types';
+
+class RestDataAPI {
+  public static save(restaurant: Restaurant): void {
+    const restaurants = this.getRestaurantsList();
+    restaurants.push(restaurant);
+    this.saveRestaurantsList(restaurants);
```

Comment:
저는 개인적으로 이렇게 사용하는 편이긴 합니다 ㅎㅎ 
```suggestion
    const restaurants = [...this.getRestaurantsList(), restaurant];
    this.saveRestaurantsList(restaurants);
```

Reply:
전개 연산자를 사용이 아직 익숙치 않은 것 같아요! 감사합니다